working-together/README.md

```diff
@@ -156,6 +156,11 @@ You can [download a double-sided printout] of these guides to display in your me
   - Let everyone know ahead of time what they can expect.
   - Create a calendar invite with links to the video call and dial-in information (if applicable)
   - Create a meeting agenda or note about what the meeting will cover.
+  - Strongly avoid vague meeting invitations. For people with anxiety, vague
```

Comment:
I wonder if we need the clarification "for people with anxiety"? I think that any vague meeting invitations — where the recipient doesn't already expect it because you might've spoken about it previously — can be quite anxiety inducing for all. What do you think?

Reply:
Hmmm, good point. I'd like to draw attention to anxiety (in the clinical sense) in particular; maybe:

> Strongly avoid vague meeting invitations. These can be stress-inducing for anyone; and for people with anxiety, they can cause panic attacks.

Reply:
Oh yes! That's great!

Reply:
I don't have anxiety around meetings and also I think people should avoid vague meeting invitations for personal reasons.

Specifically: I avoid meetings when I can, so if the meeting invitation is vague then I need to do extra work to learn whether I can skip it. Save me some time, describe your meetings.

Reply:
I feel attacked, @mike-burns, and I'm here for it!

Reply:
Always happy to chat!